src/pc/sdl2.c

```diff
@@ -164,6 +169,35 @@ u_long MyPadRead(int id) {
             pressed |= PAD_R1;
         }
 
+        if (joystick) {
+            SDL_JoystickUpdate();
+            pressed |= SDL_JoystickGetButton(joystick, 0) ? PAD_CROSS : 0;
```

Comment:
I think you're supposed to use these?
https://github.com/davidsiaw/SDL2/blob/6ecaa6b61372e5b2f9bd01201814d07e34bb4186/include/SDL_gamecontroller.h#L311

Reply:
Weird, they do not match? Only the first four fields A, B, X and Y matches. Everything else is off. Maybe it's my configuration only. Do you have any joystick you can test this with?

Reply:
I didn't test this, I just meant it seems like we should use the enum rather than magic numbers?

Reply:
Reading this it seems like the better API is SDL_GameControllerGetButton?  The enums are for that and not Joystick. https://discourse.libsdl.org/t/difference-between-joysticks-and-game-controllers/24028